tests/foreman/ui/test_modulestreams.py

```diff
@@ -39,7 +39,7 @@ def module_yum_repo(module_product):
         name=gen_string('alpha'),
         product=module_product,
         content_type='yum',
-        url=CUSTOM_MODULE_STREAM_REPO_2,
+        url=settings.repos.module_stream_1.url,
```

Comment:
`module_stream_1` equates to `CUSTOM_MODULE_STREAM_REPO_2`?

Reply:
Yeah, I renamed `CUSTOM_MODULE_STREAM_REPO_1` to `module_stream_0` and `CUSTOM_MODULE_STREAM_REPO_2` to `module_stream_1`. This closer resembles our naming convention for yum and puppet repos (`yum_0` and `puppet_0`)